lib/schema-evolution-manager/apply_util.rb

```diff
@@ -35,6 +36,24 @@ def apply!(dir)
       count
     end
 
+    def persist_pgpass(password)
+      Preconditions.check_not_blank(password, "password cannot be blank")
+      puts "Creating temp pgpass at #{@pgpass_file.path}"
+      FileUtils.chmod(0600, @pgpass_file.path)
+
+      FileUtils.rm_rf("#{ENV['HOME']}/.pgpass")
+      FileUtils.ln_s(@pgpass_file.path, "#{ENV['HOME']}/.pgpass")
```

Comment:
agree with sean's comments to use the PGPASSFILE env variable to point to our temp file